src/node/module.ts

```diff
@@ -110,8 +124,40 @@ export const builtinModules = [
 ];
 Object.freeze(builtinModules);
 
+export function findSourceMap(): void {
+  // Not meaningful to implement in the context of workerd.
+  throw new ERR_METHOD_NOT_IMPLEMENTED('module.findSourceMap');
+}
+
+export function register(): void {
+  // Not meaningful to implement in the context of workerd.
+  throw new ERR_METHOD_NOT_IMPLEMENTED('module.register');
+}
+
+export function syncBuiltinESMExports(): void {
+  // Not meaningful to implement in the context of workerd.
+  throw new ERR_METHOD_NOT_IMPLEMENTED('module.syncBuiltinESMExports');
+}
+
+//
+// IMPORTANT NOTE!
+//
+// We are deliberately not including any of these functions below because
+// they are either experimental or in active development as of January 2025.
```

Comment:
I would say "either experimental, in active development, or not relevant..."

For instance, I do not imagine that would would ever implement `registerHooks()`, `module.constants.compileCacheStatus`, `flushCompileCache`, or `getCompileCacheDir` in workers.